tbx/static_src/sass/components/_footer.scss

```diff
@@ -96,6 +92,28 @@
         }
     }
 
+    &__company {
+        display: flex;
+        flex-direction: column;
+        height: 100%;
+        margin: $spacer-medium 0 $spacer-mini;
+        color: var(--color--grey-20);
+
+        @include media-query(large) {
+            justify-content: center;
+            margin: 0;
+        }
+    }
+
+    &__company-address {
+        font-weight: 600;
+
+        > * {
```

Comment:
nitpick: I'd normally discourage generic child selectors like this, but as it only affects 2 elements I'm not too worried.